lib/bolt/project.rb

```diff
@@ -167,5 +167,131 @@ def check_deprecated_file
         Bolt::Logger.deprecation_warning('Using project.yaml instead of bolt-project.yaml', msg)
       end
     end
+
+    def migrate_project(configured_inventory_path, outputter)
+      migrating_inv = configured_inventory_path || inventory_file
+      inventory_1_to_2(migrating_inv, outputter) if migrating_inv.file?
+      bolt_yaml_to_bolt_project(migrating_inv, outputter)
+    end
```

Comment:
This needs to return a code (`0` or `1`), otherwise you'll get an error after migrating:

```
Traceback (most recent call last):
	5: from /Users/tom/.rvm/gems/ruby-2.6.3/bin/ruby_executable_hooks:24:in `<main>'
	4: from /Users/tom/.rvm/gems/ruby-2.6.3/bin/ruby_executable_hooks:24:in `eval'
	3: from /Users/tom/.rvm/gems/ruby-2.6.3/bin/bolt:23:in `<main>'
	2: from /Users/tom/.rvm/gems/ruby-2.6.3/bin/bolt:23:in `load'
	1: from /Users/tom/git/bolt/exe/bolt:11:in `<top (required)>'
/Users/tom/git/bolt/exe/bolt:11:in `exit': no implicit conversion from nil to integer (TypeError)
```

We'll likely want to have both `inventory_1_to_2` and `bolt_yaml_to_bolt_project` return a code indicating whether it was successful, and then use that to determine whether the migration overall was successful.